standard-cluster-services/src/main/java/org/terracotta/entity/IEntityMessenger.java

```diff
@@ -39,6 +39,18 @@
   void messageSelf(EntityMessage message) throws MessageCodecException;
 
   /**
+   * Defer a message based on another message, which will be sent in the future.
```

Comment:
I don't really understand *why* someone would want to do this. Can you give an example please?